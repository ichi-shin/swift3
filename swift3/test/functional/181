#!/bin/bash
# Copyright(c)2014 NTT corp. All Rights Reserved.
# DB1-1-4-3_5

. ./common
S3USER=tester

echo '<CreateBucketConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
<LocationConstraint></LocationConstraint>
</CreateBucketConfiguration >' > $tmp.xml

_s3_put /bucket -T $tmp.xml -D - | _filter_curl xml
_s3_get /bucket -D - | _filter_curl xml
