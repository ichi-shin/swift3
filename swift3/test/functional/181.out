QA output created by 181
> s3curl --id tester -- -X PUT -T /TMP.xml -D - http://SWIFT_HOST/bucket... 400
HTTP/1.1 100 Continue

HTTP/1.1 400 Bad Request
Content-Type: text/xml
Date: DATE
Transfer-Encoding: chunked
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<Error>
  <Code>InvalidLocationConstraint</Code>
  <Message>The specified location constraint is not valid.</Message>
  <RequestId>TXID</RequestId>
</Error>
> s3curl --id tester -- -X GET -D - http://SWIFT_HOST/bucket... 404
HTTP/1.1 404 Not Found
Content-Type: text/xml
Date: DATE
Transfer-Encoding: chunked
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<Error>
  <Code>NoSuchBucket</Code>
  <Message>The specified bucket does not exist.</Message>
  <RequestId>TXID</RequestId>
  <BucketName>bucket</BucketName>
</Error>
